docker-compose.yml

```diff
@@ -58,6 +58,25 @@ services:
     ports:
       - "0.0.0.0:6379:6379"
 
+  celery:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: celery -A fidesops.tasks beat -l info --logfile=/var/log/celery.log
```

Comment:
This works in development, but for a production deployment we need to make the celery worker configuration more foolproof

Can we add a new CLI command like `fidesops worker`, which acts similarly to `fidesops webserver` except it starts celery instead of uvicorn?

Reply:
In the interest of [unblocking the DRP revoke / cancel request feature](https://github.com/ethyca/fidesops/issues/525) I've added [this follow-up ticket](https://github.com/ethyca/fidesops/issues/663) for this suggestion